CHANGELOG.md

```diff
@@ -9,6 +9,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfluff/sqlfluff/releases) and will be copied in here on each release (please remember to update the issues and contributors to links!). There is no need to manually edit this file going forward.
 -->
 
+## [0.8.0] - 2021-11-06
```

Comment:
Maybe go tomorrow to give people the chance to merge those last few PRs? Let's play it by ear and see.

```suggestion
## [0.8.0] - 2021-11-07
```